modules/apps/portlet-configuration/portlet-configuration-web/src/main/resources/META-INF/resources/edit_permissions.jsp

```diff
@@ -345,17 +351,20 @@ PortletConfigurationPermissionPropagation portletConfigurationPermissionPropagat
 					displayType: 'secondary',
 					label: '<liferay-ui:message key="discard" />',
 					onClick: () => {
-						window.location.href = href;
+						Liferay.Util.navigate(href);
 					},
 				},
 				{
 					displayType: 'warning',
 					label: '<liferay-ui:message key="save-and-continue" />',
 					onClick: () => {
-						<portlet:namespace />saveButton.dispatchEvent(
-							new Event('click')
+						saveForm();
+
+						var redirect = document.getElementById(
```

Comment:
it should be add before saveForm();. It is easy to understand code

Reply:
Fixed it. Thanks

Reply:
Fixed it. Thanks